packages/input/src/interfaces.ts

```diff
@@ -50,4 +52,6 @@ export interface ISubComponentProps extends IIsRTL {
   invalid?: boolean;
   disabled?: boolean;
   css?: any;
+  hasValue?: boolean;
+  focusOnInput?: boolean;
```

Comment:
are you sure this is needed here? This type defines the public properties of the input, not the inside